resources/eventing/values.yaml

```diff
@@ -9,7 +9,7 @@ global:
       pullPolicy: "IfNotPresent"
     publisher_proxy:
       name: event-publisher-proxy
-      version: v20221230-5e8bc6ca
+      version: v20230111-18887898
```

Comment:
lets keep it same as `v20221230-5e8bc6ca`, and once this PR is merged then bump the image. How did you get a version before PR is merged?

Reply:
good point! It is the build of main, which shouldn't be released.